pkg/cmd/runsGet.go

```diff
@@ -87,12 +88,14 @@ func executeRunsGet(cmd *cobra.Command, args []string) {
 	var console = utils.NewRealConsole()
 
 	apiServerUrl := bootstrapData.ApiServerURL
-	log.Printf("The API sever is at '%s'\n", apiServerUrl)
+	log.Printf("The API server is at '%s'\n", apiServerUrl)
+
+	apiClient := auth.GetAuthenticatedAPIClient(apiServerUrl, fileSystem, galasaHome)
```

Comment:
surely this can fail ?

Reply:
See https://github.com/galasa-dev/cli/pull/151#discussion_r1328685969